src/arrays.js

```diff
@@ -8,33 +8,60 @@ const each = (elements, cb) => {
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
   // based off http://underscorejs.org/#each
+  for (let i = 0; i < elements.length; i++) {
+    cb(elements[i], i);
```

Comment:
No need to pass the index `i` here; we don't know what the callback function's signature (the number and types of inputs it accepts) is, so err on the side of just passing in `elements[i]`.